tests/sentry/migrations/test_0001_emerge_upload_models.py

```diff
@@ -0,0 +1,18 @@
+from sentry.testutils.cases import TestMigrations
+
+
+class AddPreprodModelsTests(TestMigrations):
```

Comment:
You don't need to include migration tests for schema modifications. We only need migration tests for backfills or data mutations.

Reply:
Yea this one felt a bit silly to write, but wanted to follow "For this reason, every migration should have a corresponding integration test." from the docs lol

Reply:
also this test is in the wrong place -- this is for *sentry*'s migration tests but you have a separate application so those should live there

generally the recommendation from the docs is for nontrivial / data migrations